app/uk/gov/hmrc/cdsreimbursementclaim/services/ClaimTransformerService.scala

```diff
@@ -131,7 +144,8 @@ class DefaultClaimTransformerService @Inject() (uuidGenerator: UUIDGenerator, da
 
     (
       makeReasonAndOrBasisOfClaim(
-        completeClaim.maybeBasisOfClaimAnswer
+        completeClaim.maybeBasisOfClaimAnswer,
+        getFeatureFlag[Boolean]("enable-correct-additional-information-code-mapping")
```

Comment:
Repeating this string is problematic. Please move it into a val somewhere.
Or maybe make a method:
```
def getCorrectAdditionalInformationCodeMapping():Boolean = 
   getFeatureFlag[Boolean]("enable-correct-additional-information-code-mapping")
```

Reply:
done